### PR TITLE
Cleanup/update dependencies 

### PR DESCRIPTION
**Pull request by mnatsheh**

### CODE REVIEW

**Review by IslamJMomani — APPROVED** [review]

File: src/components/Comments/CommentBody.tsx [hunk 1]
```diff
@@ -17,12 +17,12 @@ import AnnotationReply from '@manuscripts/assets/react/AnnotationReply'
 import { Keyword, UserProfile } from '@manuscripts/json-schema'
 import { Field, FieldProps, Form, Formik } from 'formik'
 import React, { Dispatch, SetStateAction, useEffect } from 'react'
-import ReactTooltip from 'react-tooltip'
 import styled from 'styled-components'
 
 import { CommentType, UnsavedComment } from '../../lib/comments'
 import { ButtonGroup, PrimaryButton, SecondaryButton } from '../Button'
 import { FormError } from '../Form'
+import { Tooltip } from '../Tooltip'
```
Comment (mbartenev-atypon) [inline src/components/Comments/CommentBody.tsx:25]: As I mentioned on article editor. Please separate cleanup into a different PR. Thanks

**Review by mbartenev-atypon — CHANGES_REQUESTED** [review]

File: src/components/Comments/CommentBody.tsx (hunk 1 above)
Comment (mnatsheh) [reply src/components/Comments/CommentBody.tsx:25]: I don't think this is the same. Whereas in article-editor, I can see how someone can get confused by why some of the changes are there, I think these changes fit this PR; there are no business logic changes.

**Review by mbartenev-atypon — APPROVED** [review]

**Review by asouqi — APPROVED** [review]
